syntax = "proto3";

message Input {
    string url = 1;
    float start_time_sec = 2;
    float stop_time_sec = 3;
}

message Output {
    string value = 1;
}

service VideoCaptioning {
    rpc video_cap(Input) returns (Output) {}
}